controls/raddiagram/diagram-items/features-container-shapes.md

```diff
@@ -242,36 +242,81 @@ Below you can find a list of all __RadDiagramContainerShape__ members that are r
+* __Cut and Copy__: These clipboard operations work only on the __ContainerShape__. The shapes inside the container won't be cut or copied. You can find more information about the clipboard operations supported in the __RadDiagram__ in [this tutorial]({%slug raddiagrams-features-clipboard%}).			
 
 If you do wish to rotate, scale, cut or copy both the container and its children simultaneously, you can do so by dragging a selection rectangle around the container (instead of just clicking-selecting the container). This selection will contain both the container and the children thus allowing you to perform the aforementioned actions on all items at the same time.  
 
 ## Customize the ContainerShape Appearance
 
 You can easily customize the visual appearance of the __RadDiagramContainerShape__ by using the following properties:		
 
-* __Background__ - gets or sets the brush that specifies the __RadDiagramContainerShape__ background color.			
+* __Background__: Gets or sets the brush that specifies the __RadDiagramContainerShape__ background color.			
 
-* __BorderBrush__ - gets or sets the brush that specifies the __RadDiagramContainerShape__ border color.			
+* __BorderBrush__: Gets or sets the brush that specifies the __RadDiagramContainerShape__ border color.			
 
-* __BorderThickness__ - gets or sets the width of the __RadDiagramContainerShape__ outline.			
+* __BorderThickness__: Gets or sets the width of the __RadDiagramContainerShape__ outline.			
 
 >tip You can use the __RadDiagram.ContainerShapeStyle__ property to explicitly apply a style on all __RadDiagramContainerShapes__ in a __RadDiagram__ instance.		  
 
+## Container Service
+
+* __CompleteManipulation__: A void method that completes given manipulation. Accept parameter of type __CompositeAsyncStateCommand__.
+* __DetachItemsFromContainer__: A void method that detaches items from a container.
+* __GetChildrenBounds__: This method returns Rect element which wraps the items inside the container.
+* __GetContainerChildrenRecursively__: This method search recursively and return all children of the container.
+* __GetFloatingChildren__: This method return all shapes and connection which are not attched to shapes.
+* __GetParentContainersRecursively__: This method search recursively and return all parent containers.
+* __GetRootItems__: A void method the return all root containers if any exist. 
+* __UpdateContainerLayout__: A void method thats update the container layout. Accept parameter of type __IDragDropAware__.
+
+## How To
+
+* __Prevent dropping a shape in container__: To prevent dropping a shape inside a container, you can set the __IsDropEnabled__ property of the __RadDiagramContainerShape__ to __False__.
+
+## IDragDropAware interface

```

Comment:
The section name doesn't mean much to the user. Consider using more descriptive naming. For example, "Customize drag and drop actions"